nflow-engine/src/main/java/com/nitorcreations/nflow/engine/listener/WorkflowExecutorListener.java

```diff
@@ -85,6 +85,28 @@ public ListenerContext(AbstractWorkflowDefinition<?> definition, WorkflowInstanc
   void beforeProcessing(ListenerContext listenerContext);
 
   /**
+   * Processing chain.
+   * Process methods in listeners form a
```

Comment:
Could WorkflowExecutorListener interface contain also appliesTo(WorkflowDefinition) method so that each process-method implementation wouldn't need separate if check, if the listener applies to only certain workflow definitions?